manager.py

```diff
@@ -156,33 +180,57 @@ def fund_distributor(btc_amount):
         sleep(1)
     print(f"- funded (current balance {balance / BTC:.8f} BTC)")
 
+def create_rpc_client(client_version, ip, port, name, delay, skip_rounds):
```

Comment:
I propose renaming this function to `create_wasabi_client` or `init_wasabi_client` as we plan on integrating other clients too.
```suggestion
def init_wasabi_client(client_version, ip, port, name, delay, skip_rounds):
```